addon/session-stores/cookie.js

```diff
@@ -148,6 +161,8 @@ export default BaseStore.extend({
   init() {
     this._super(...arguments);
 
+    this.set('_cookiePath', Configuration.baseURL);
```

Comment:
This should only be set when it is `null` (as it might have been set when the cookie store instance was created with `Cookie.create({ cookiePath: '/whatever' })` and shouldn't be overridden in this case).